### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Bpotiaty**
- [DEMO LINK](https://bpotiaty.github.io/layout_stars/)
- [TEST REPORT LINK](https://bpotiaty.github.io/layout_stars/report/html_report/)

- [x] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [x] Each BEM block has its own separate file
- [x] All Typical Mistakes from BEM lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/Bpotiaty/layout_stars/blob/master/checklist.md)

### CODE REVIEW

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, remove margin-right from the last stars in a row
image

File: src/style.css [hunk 1]
```diff
@@ -1 +1,31 @@
...
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars .stars__star:last-child {
+  margin-right: 0;
+}
```
Comment (nazarmatsevych) [inline src/style.css:31]:
```suggestion
.stars__star:last-child {
  margin-right: 0;
}
```

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Did you rerun npm run deploy?

If you have any questions - feel free to ask them in fe_chat

File: src/style.css (hunk 1 above)
Comment (Bpotiaty) [reply src/style.css:31]: Unfortunately I didn't  rerun npm run deploy. Correcting myself)

**Pull request closed without merging**